battleship.py

```diff
@@ -192,6 +193,8 @@ def getClickedCell(data, event):
 Returns: None
 '''
 def drawShip(data, canvas, ship):
+    for row in ship:
```

Comment:
To improve the readability you can collect the coordinates in the variable and pass it in the function